DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
```

Comment:
This is really baking my noodle. We've had `ExternalLoginButtonsPartial`, `ExternalLoginMethodsPartial`, `ExternalLoginsListPartial` (deleted) and now `ExternalLoginsPartial`. All of them under /Views/Account. What do these things do, where do they appear, etc., etc.?